editor/src/core/model/element-template-utils.spec.tsx

```diff
@@ -1954,3 +1957,56 @@ describe('transformJSXComponentAtPath', () => {
     expect((updatedElement.children[0] as JSXTextBlock).text).toEqual('hello')
   })
 })
+
+describe('renameJSXElementChild', () => {
+  it('should rename a simple element', () => {
+    const jsxElementToRename = jsxElement('Flex', 'flex-1', [], [])
```

Comment:
ultra nitpick, but `flex-1` would be an illegal javascript name which would throw an exception :)